lib/manageiq/appliance_console/message_configuration.rb

```diff
@@ -0,0 +1,134 @@
+require 'active_support/core_ext/module/delegation'
+require 'pathname'
+
+module ManageIQ
+  module ApplianceConsole
+    class MessageConfiguration
+      attr_reader :username, :password,
+                  :miq_config_dir_path, :config_dir_path, :sample_config_dir_path,
+                  :tools_log4_properties_path, :tools_log4_properties_sample_path,
+                  :client_properties_path,
+                  :keystore_dir_path, :truststore_path, :keystore_path,
+                  :messaging_yaml_sample_path, :messaging_yaml_path
+
+      BASE_DIR                          = "/opt/kafka".freeze
+      LOGS_DIR                          = "#{BASE_DIR}/logs".freeze
```

Comment:
Do you know what's put here?  If it's server logs, we should probably move that to `/var/log`, but if it's message data that they're calling "logs" it's fine to stay here.

Reply:
@bdunne 

It seems to be server logs. I'm OK with reconfiguring Kafka to write to `/var/log` but it seems kafka's logging configuration is a bit complex so I if we decide these logs really should be relocated I'd like to tackle that in a follow on PR.
Are you OK with that?

Reply:
Ok, followup sounds good.